package.json

```diff
@@ -53,11 +53,12 @@
     "eslint-plugin-react": "^5.2.2",
```

Comment:
So when I run `npm test` it goes through the tests, but just hangs:

![image](https://user-images.githubusercontent.com/794517/47822209-ea0dbf80-dd28-11e8-85e3-93bd6af5db10.png)

If you try in master (you'll need to `npm install` again), it immediately exits after displaying the results.

(btw, ignore the 5 failing tests. I'm planning to fix those soonish.)